frontend/src/metabase/entities/questions.js

```diff
@@ -77,6 +81,17 @@ const Questions = createEntity({
   },
 
   reducer: (state = {}, { type, payload, error }) => {
+    if (type === SOFT_RELOAD_CARD) {
+      const { id } = payload;
+      const latestReview = payload.moderation_reviews?.find(x => x.most_recent);
+
+      if (latestReview) {
+        return updateIn(state, [id], q => ({
+          ...q,
+          moderated_status: latestReview.status,
+        }));
```

Comment:
Let's maybe use `question` instead of just `q` here? It might not be immediately obvious what it means